### PR TITLE
(Des)integrate visionneuse into repondeur

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: repondeur/zam_repondeur/models/visionneuse.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+@dataclass
+class Article:
+    pk: str
+    id: int
+    titre: str
+    etat: Optional[str] = ""
+    multiplicatif: Optional[str] = ""
+    jaune: Optional[str] = ""
+    alineas: Optional[dict] = None
+    amendements: Any = field(default_factory=lambda: [])  # List[Amendement]
```
Comment (ronnix) [inline repondeur/zam_repondeur/models/visionneuse.py]: Why the convoluted type annotation here?

Comment (davidbgk) [reply repondeur/zam_repondeur/models/visionneuse.py]: Because you cannot reference `Amendement` which is declared below.

Comment (ronnix) [reply repondeur/zam_repondeur/models/visionneuse.py]: Try putting in in double quotes, mypy should handle that.

**Comment by davidbgk** [comment]
`−1,121` even better \o/

**Pull request merged**